syntax = "proto3";

package metrics;

service MetricsService {
  // Un flujo (stream) para enviar m√©tricas de forma eficiente
  rpc SendMetric (MetricRequest) returns (MetricResponse);
}

message MetricRequest {
  string service_name = 1;
  string metric_name = 2;
  double value = 3;
  string metric_type = 4;
  map<string, string> labels = 5;
}

message MetricResponse {
  bool success = 1;
  string message = 2;
}